torch/quantization/fx/quantize.py

```diff
@@ -249,21 +249,23 @@ def node_arg_is_weight(node: Node, arg: Any) -> bool:
                 return True
     return False
 
-# A dictionary for querying the weight index for a given op
-# TODO(future PR): handle linear
-BIAS_INDEX_DICT = {
-    torch.nn.functional.conv1d : [2],
-    torch.nn.functional.conv2d : [2],
-    torch.nn.functional.conv3d : [2],
+CONV_OPS_WITH_BIAS = {
+    torch.nn.functional.conv1d,
+    torch.nn.functional.conv2d,
+    torch.nn.functional.conv3d,
 }
+CONV_BIAS_ARG_INDEX = 2
 
 def node_arg_is_bias(node: Node, arg: Any) -> bool:
-    if isinstance(node, Node) and node.op == 'call_function' and \
-            node.target in BIAS_INDEX_DICT:
-        for i, node_arg in enumerate(node.args):
-            if arg is node_arg and i in \
-                    BIAS_INDEX_DICT[node.target]:  # type: ignore
-                return True
+    if isinstance(node, Node) and node.op == 'call_function':
+        if node.target in CONV_OPS_WITH_BIAS:
+            for i, node_arg in enumerate(node.args):
+                if arg is node_arg and i == CONV_BIAS_ARG_INDEX:
+                    return True
+        elif node.target is torch.nn.functional.linear:
+            for kwarg_name, kwarg_value in node.kwargs.items():
+                if kwarg_name == 'bias' and arg is kwarg_value:
```

Comment:
hmmm, can 'weight' be a kwarg as well?

Reply:
yeah, `weight` is a keyword argument for Linear: https://pytorch.org/docs/stable/nn.functional.html#torch.nn.functional.linear

Reply:
Bias isn't kwarg-only argument (e.g. the following code is valid `torch.nn.functional.linear(x,weight, bias) `) - will this situation still be correctly handled?

Reply:
Yeah, this works with the current code.  The test case added in this PR traces through an instance of `nn.Linear`, which contains the call to `F.linear(input, self.weight, self.bias)`, with bias specified as a positional argument.

I believe the reason that it works is that when FX traces through the calls, the arguments which are defined in the function def as keyword are captured in `node.kwargs`, even if they are specified by the user as positional.  I haven't directly verified this yet.